app/uk/gov/hmrc/eoricommoncomponent/frontend/domain/messaging/subscription/ContactInformation.scala

```diff
@@ -88,7 +88,7 @@ object ContactInformation extends CommonHeader {
           city = Some(fourLineAddress.city),
           postalCode = fourLineAddress.postcode,
           countryCode = Some(fourLineAddress.countryCode),
-          telephoneNumber = Some(contactDetails.telephone),
+          telephoneNumber = if (contactDetails.telephone.isBlank) None else Some(contactDetails.telephone),
```

Comment:
It does yep - phoneNumber in the sub02 schema is an optional field which when exists requires to be between 1-24 numbers. This change was added in to set the value to None because we would otherwise send an empty String which would fail the length validation. By setting it to None we don't send this at all